spec/features/step_definitions/web_steps.rb

```diff
@@ -0,0 +1,219 @@
+# Taken from the cucumber-rails project.
```

Comment:
this is a cucumber-sinatra init generated file - remove it as you are not using cucumber